pkg/storage/mongo/mongo.go

```diff
@@ -4,32 +4,29 @@ import (
 	"github.com/globalsign/mgo"
 )
 
+const (
+	coll = "modules"
+	db   = "athens"
+)
+
 // ModuleStore represents a mongo backed storage backend.
 type ModuleStore struct {
-	s   *mgo.Session
-	d   string // database
-	c   string // collection
-	url string
+	s     *mgo.Session
+	deets *ConnDetails
+	d     string // database
+	c     string // collection
+	url   string
 }
 
 // NewStorage returns an unconnected Mongo backed storage
 // that satisfies the Backend interface.  You must call
 // Connect() on the returned store before using it.
```

Comment:
update comment pls. you dont have Connect capability now

Reply:
will do, thanks @michalpristas